src/assets/scripts/client/airport/AirportWind.js

```diff
@@ -0,0 +1,35 @@
+import AirportModel from './AirportModel';
+import { calculateNextWind } from '../utilities/windUtilities';
+
+/**
+ * Mutable array that holds the information on the airport's current wind.
+ */
+export let airportWind = {
+   speed: 10,
+   angle: 0
+};
+
+export default class AirportWind {
+    /**
+     * Constructor
+     */
+    constructor() {}
+
+    /**
+     * Sets the initial airportWind, instead of the default.
+     * Also can be used to force-set the wind.
+     * 
+     * @method setStaticWind
+     * @param nextWind {Array} [speed, angle]
+     */
+    setStaticWind(nextWind) {
+        airportWind = nextWind;
```

Comment:
now you can use the class properties instead of updating the object.

that is an object, by the way, not an array.  you can also use `const` with objects even if the internal properties change.  as long as the reference to the object does not change, `const` is still good.